src/ds/ds5/ds5-private.cpp

```diff
@@ -0,0 +1,14 @@
+//// License: Apache 2.0. See LICENSE file in root directory.
+//// Copyright(c) 2015 Intel Corporation. All Rights Reserved.
+
+#include "ds5-private.h"
+
+using namespace std;
+
+namespace librealsense
+{
+    namespace ds
+    {
+        
+    } // librealsense::ds
+} // namespace librealsense
```

Comment:
We can place here ds5 specific data, otherwise it is not needed.. right?

Reply:
done